src/Tizen.NUI/src/public/BaseComponents/TextField.cs

```diff
@@ -1101,6 +1101,22 @@ public bool EnableGrabHandlePopup
             }
         }
 
+        /// <summary>
+        /// The Selected Text property.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done, Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string SelectedText
+        {
+            get
+            {
+                string temp;
+                GetProperty(TextField.Property.SELECTED_TEXT).Get(out temp);
+                return (string)temp;
```

Comment:
You don't need to cast the string to string again.

Reply:
thank you, cast is removed